Running: /home/sarnold/bin/audit-code.sh -c python ./
Scanning './'
== subprocess_spawned() ==
./setup.py:16:    exec(f.read(), about)

== memory_management() ==
(unsupported type 'python')

== files() ==
./setup.py:15:with open(os.path.join(base_dir, "cryptography_vectors", "__about__.py")) as f:
./cryptography_vectors/__init__.py:23:    return open(os.path.join(base, filename), mode)

== logging() ==

== environment() ==

== privileged() ==

== networking() ==
./setup.py:16:    exec(f.read(), about)

== http:// ==

== encryption ==
./cryptography_vectors/__about__.py:15:__uri__ = "https://github.com/pyca/cryptography"

== sql() ==

== tmp() ==

== priv_cmds (sudo, gksu, pkexec) ==

== envvars() ==

== webkit() ==

== comments (XXX, FIXME, TODO) ==

== unsafe input mechanisms ==
(unsupported type 'python')

